File: outros/ws - pycharm/primeiro/Retangulo.py
import math

base = float(input("Base do retangulo: "))
altura = float(input("Altura do retangulo: "))

area = base * altura
perimetro = base * 2 + altura * 2
diagonal = math.sqrt(base * base + altura * altura)

print(f"Area = {area:.4f}")
print(f"Perimetro = {perimetro:.4f}")
print(f"Diagonal = {diagonal:.4f}")

File: exercicios/c, c++/c/exercicios/retangulo.c
#include <stdio.h>
#include <math.h>


int main()
{
    double base, height, area, perimeter, diagonal;

    printf("What is the base of the rectangle: ");
    scanf("%lf", &base);
    printf("What is the height of the rectangle: ");
    scanf("%lf", &height);

    area = base * height;
    perimeter = (base * 2) + (height * 2);
    diagonal = sqrt(base * base + height * height);

    printf("Area = %.4lf\n", area);
    printf("Perimeter = %.4lf\n", perimeter);
    printf("Diagonal = %.4lf\n", diagonal);

    return 0;
}
